src/ObjectHelper.php

```diff
@@ -50,6 +50,16 @@ public function buildTransaction(stdClass $object)
             }
         }
 
+        // Vindicia screw up the paypal email field by a camel case typo
```

Comment:
Can you please reword this to something like
```
There is a discrepancy between Vindicia document and actual Vindicia response regarding 'paypalEmail`` field
```

Reply:
and can you say which is which?